typing_extensions/src_py3/test_typing_extensions.py

```diff
@@ -0,0 +1,624 @@
+import sys
+import os
+import abc
+import contextlib
+import collections
+from unittest import TestCase, main, skipUnless
+from typing import TypeVar, Optional
+from typing import T, KT, VT  # Not in __all__.
+from typing import Tuple, List
+from typing import Generic
+from typing import get_type_hints
+from typing import no_type_check
+from typing_extensions import NoReturn, ClassVar, Type, NewType
+import typing
+import typing_extensions
+import collections.abc as collections_abc
+
+TYPING_LATEST = sys.version_info[:3] < (3, 5, 0)
```

Comment:
I am not sure I follow the logic here, shouldn't this be ``TYPING_OLDEST``? Or this is the version of typing you "emulate"? Could you please add a comment here?